components/dialog/dialog-mixin.js

```diff
@@ -407,8 +407,10 @@ export const DialogMixin = superclass => class extends RtlMixin(superclass) {
 			this._height = this._getHeight();
 			await this.updateComplete;
 		}
-		this._updateOverflow();
-		await this.updateComplete;
+		requestAnimationFrame(async() => {
```

Comment:
It looks like it won't make any difference to the current callers, however if we want the resulting `Promise` (returned by `_updateSize`) to still resolve when the update is complete, I think we would need to wrap the rAF in a `Promise` and either `return` it, or `await` it.  None of the callers actually rely on the `Promise` that's returned though... the method is only really marked as `async` in order to use `await` internally.

```javascript
...
...
await new Promise(resolve => {
    requestAnimationFrame(async() => {
        this._updateOverflow();
        await this.updateComplete;
        resolve();
    });
});
```